projects/ngx-meta/schematics/ng-add/index.ts

```diff
@@ -1,23 +1,31 @@
 import { chain, noop, Rule } from '@angular-devkit/schematics'
 import { addRootProvider } from '@schematics/angular/utility'
-import { Schema } from './schema'
+import { MetadataModules, Schema } from './schema'
 import { classify } from '@angular-devkit/core/src/utils/strings'
 
+const ENTRYPOINTS = new Set<MetadataModules>([
+  'json-ld',
+  'open-graph',
+  'standard',
+  'twitter-card',
+])
 // noinspection JSUnusedGlobalSymbols (actually used in `collection.json`)
 export function ngAdd(options: Schema): Rule {
-  const maybeAddNgxMetaRootProvider = (name?: string): Rule => {
-    if (!name) {
-      return noop()
-    }
+  const addNgxMetaRootProvider = (name: string): Rule => {
+    const entrypoint =
+      [...ENTRYPOINTS].find((entrypoint) => name.startsWith(entrypoint)) ?? name
     return addRootProvider(
       options.project,
       ({ code, external }) =>
-        code`${external(`provideNgxMeta${classify(name)}`, `@davidlj95/ngx-meta/${name}`)}()`,
+        code`${external(`provideNgxMeta${classify(name)}`, `@davidlj95/ngx-meta/${entrypoint}`)}()`,
     )
   }
 
   return chain([
-    maybeAddNgxMetaRootProvider('core'),
-    maybeAddNgxMetaRootProvider(options.routing ? 'routing' : undefined),
+    addNgxMetaRootProvider('core'),
+    options.routing ? addNgxMetaRootProvider('routing') : noop(),
```

Comment:
Doing the `maybeAddNgxMetaProvider` was an early abstraction. Turns out only useful for routing case. Soo adding the if/else here only.